console/src/components/Services/Settings/MetadataOptions/MetadataOptions.js

```diff
@@ -3,11 +3,13 @@ import ExportMetadata from './ExportMetadata';
 import ImportMetadata from './ImportMetadata';
 import ReloadMetadata from './ReloadMetadata';
 import ResetMetadata from './ResetMetadata';
+import ExportSchema from './ExportSchema';
 
-const MetadataOptions = props => {
-  const styles = require('../Settings.scss');
+import styles from '../Settings.scss';
+import { exportSchema } from '../Actions';
 
-  const getMetadataImportExportSection = () => {
+const MetadataOptions = props => {
+  const MetadataImportExportSection = () => {
```

Comment:
Can you move it outside of the component? Having components inside of components causes render issues as React will be unable to reconcile the rendering of these components, as they are treated as new components every time.